.github/workflows/tests-e2e.yml

```diff
@@ -0,0 +1,59 @@
+name: Tests e2e
+
+on:
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  build:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        node-version: [20.x]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Bun
```

Comment:
we should use pnpm as that is what the project uses

Reply:
Done! Reverted back to `pnpm` and updated it to latest version